content/v2.0/reference/cli/influx/config/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: influx config – Config management commands
```

Comment:
- ~– Config management commands~

I went through and removed all these "summaries" in the titles on the beta-6 branch. There weren't really adding anything.

Reply:
good call!